src/static/styles/partials/_skeleton.css

```
@@ -0,0 +1,296 @@
+}
+
+.progress-skeleton .skeleton-step-indicator {
+    width: 12px;
+    height: 12px;
+    border-radius: 50%;
+    background: var(--muted);
+    transition: background-color 0.3s ease;
+}
+
+.progress-skeleton .skeleton-progress-step.active .skeleton-step-indicator {
+    background: var(--primary);
```

Stylelint (17.7.0)

[error] Duplicate property "background" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)